front/src/common/StationCard.tsx

```diff
@@ -33,17 +34,15 @@ export default function StationCard({ station, onClick, fixedHeight = false }: P
     >
       <div className="station-card-head">
         <span className="station-card-code">{trigram}</span>
-        <span className="station-card-name">{name}</span>
-        {yardname && !yardNamesToExclude.includes(yardname) && (
-          <span className="station-card-ch">{yardname}</span>
-        )}
+        <span className="station-card-name">{name}&nbsp;</span>
+        {yardname && !yardNamesToExclude.includes(yardname) && <span>{yardname}</span>}
+        <span className="station-card-uic ml-3">{uic ? formatUicToCi(uic) : ''}</span>
```

Comment:
Can we hide this span if uic is not defined ?
```suggestion
        { uic && <span className="station-card-uic ml-3">{formatUicToCi(uic)}</span>
```